staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto

```diff
@@ -730,11 +730,20 @@ message SELinuxOption {
 }
 
 // Capability contains the container capabilities to add or drop
+// Dropping a capability will drop it from all sets.
+// If a capability is added to only the add_capabilities list then it gets added to permitted, 
+// inheritable, effective and bounding sets, i.e. all sets except the ambient set.
+// If a capability is added to only the add_ambient_capabilities list then it gets added to all sets, i.e permitted
+// inheritable, effective, bounding and ambient sets.
+// If a capability is added to add_capabilities and add_ambient_capabilities lists then it gets added to all sets, i.e.
+// permitted, inheritable, effective, bounding and ambient sets.
 message Capability {
     // List of capabilities to add.
     repeated string add_capabilities = 1;
     // List of capabilities to drop.
     repeated string drop_capabilities = 2;
+    // List of ambient capabilities to add.
+    repeated string add_ambient_capabilities = 3;
```

Comment:
do we have to keep updating the `v1alpha2`? Do we expect this to be implemented on Containerd 1.5, for example?

Reply:
I only expect this feature to be implemented in the latest version of containerd. I believe that would be 1.5.x wouldn't it? Sorry I am not very familiar with how CRI-API versions translate to containerd verisons.

Reply:
containerd 1.6 is about to be released. it supports CRI `v1`, which means there is no need to fall back to `v1alpha2`.

Reply:
Ack. I'll remove the v1alpha2 changes. Thanks!

Reply:
Done! PTAL. Thanks for all your help!

Reply:
@SergeyKanzhelev - looks like removing the changes from v1alpha2 causes `TestV1alpha2ContainerConfig ` test to fail. I think we will need to make the changes in v1 and v1alpha2. WDYT?